.github/workflows/ci.yml

```diff
@@ -27,66 +27,77 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0  # required for setuptools_scm
-    - name: Set up Python 3.7
-      uses: actions/setup-python@v2
+    - name: Build sdist and temporary wheel
+      run: pipx run build
+    - uses: actions/upload-artifact@v2
       with:
-        python-version: 3.7
-    - name: Install dependencies
-      run: python -m pip install build
-    - name: Build temporary sdist and wheel
-      run: python -m build
+        name: sdist
+        path: dist/*.tar.gz
 
   test:
-    timeout-minutes: 5
+    timeout-minutes: 10
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, "3.10"]
         os: [ubuntu-latest]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
         include:
-        - python-version: 3.8
-          os: macos-latest
+        - os: macos-latest
+          python-version: 3.8
+        - os: windows-latest
+          python-version: 3.8
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Install tox
       run: python -m pip install tox
     - name: Test
       run: tox -e py
     - name: Upload coverage report
       uses: codecov/codecov-action@v1
 
-  deploy:
-    timeout-minutes: 5
-    runs-on: ubuntu-latest
-    needs: [lint, test, build]
-    if: startsWith(github.ref, 'refs/tags')
+  wheels:
+#    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
Before merging, this needs to be uncommented again. I just want to see the generated wheels once.

Reply:
@marcelm. Don't forget!